### PR TITLE
XSUP-40483 - Documentation update in CS Falcon

### PR DESCRIPTION
**Pull request by RotemAmit**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/XSUP-40483)

## Description
Updated the documentation of the query params in the integration documentation following [XSUP-40483](https://jira-dc.paloaltonetworks.com/browse/XSUP-40483).

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by MLainer1 — COMMENTED** [review]
😎

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md [hunk 1]
```diff
@@ -6503,3 +6503,4 @@ Retrieve vulnerability details for a specific ID and host. Supported with the Cr
 ## Troubleshooting
 * When encountering connectivity or authorization errors within Cortex XSOAR 8, it is necessary to include the IP address corresponding to the relevant region in the CrowdStrike Falcon allow list.
 * When encountering a 429 error code returned from Crowdstrike Falcon within Cortex XSOAR 8, the solution is to use an engine as explained in this [link](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Administrator-Guide/Engines).
+* In the different query parameters such as Endpoint Detections fetch query, Endpoint Incidents fetch query, etc. if one wants to query for multiple values in the same field for example, retrieving sevirity_name that equals to Medium, High, and Critical, one needs to use the following format: field:['value1','value2','value3'].
```
Comment (ShirleyDenkberg) [inline Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md:6506]:
```suggestion
* In the different query parameters such as Endpoint Detections fetch query, Endpoint Incidents fetch query, etc. if you want to query for multiple values in the same field (for example, retrieving sevirity_name equal to Medium, High, and Critical) you need to use the following format: field:['value1','value2','value3'].
```
Is sevirity_name spelled correctly?  I think it should be severity_name.

**Comment by ShirleyDenkberg** [comment]
@MLainer1 @eyalpalo Doc review completed.

File: Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md (hunk 1 above)
Comment (RotemAmit) [reply Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md:6506]: Yes, it is severity_name. I will update the PR accordingly.

**Review by eyalpalo — APPROVED** [review]

**Pull request merged**